rtos/TARGET_CORTEX/rtx5/RTX/Source/TOOLCHAIN_GCC/TARGET_M3/irq_cm3.S

```diff
@@ -91,30 +95,25 @@ SVC_ContextRestore:
         LDR      R0,[R2,#TCB_SP_OFS]    // Load SP
         LDMIA    R0!,{R4-R11}           // Restore R4..R11
         MSR      PSP,R0                 // Set PSP
-#ifdef FEATURE_UVISOR
```

Comment:
this this one is a bit problematic for uVisor

Reply:
Thanks for reviewing the code. The suggestions is to keep this in

Reply:
correct